src/tests/ftest/util/daos_core_base.py

```diff
@@ -44,54 +48,48 @@ def __init__(self, *args, **kwargs):
             self.timeout = test_timeout
 
     def setUp(self):
+        """Set up before each test."""
         super(DaosCoreBase, self).setUp()
         self.subtest_name = self.params.get("test_name",
                                             '/run/daos_tests/Tests/*')
         self.subtest_name = self.subtest_name.replace(" ", "_")
-        logfile_env = os.environ['D_LOG_FILE']
-        self.log_dir, self.server_log = os.path.split(logfile_env)
+
+        # Determine the path and name of the daos server log using the
+        # D_LOG_FILE env or, if not set, the value used in the doas server yaml
+        self.log_dir, self.server_log = os.path.split(
+            os.getenv("D_LOG_FILE", SERVER_LOG))
         self.client_log = os.path.join(self.log_dir,
                                        self.subtest_name + "_" + CLIENT_LOG)
         # To generate the seperate client log file
         self.orterun_env = '-x D_LOG_FILE={}'.format(self.client_log)
 
     def tearDown(self):
+        """Tear down after each test."""
         super(DaosCoreBase, self).tearDown()
 
         # collect up a debug log so that we have a separate one for each
         # subtest
         if self.subtest_name:
             try:
-                new_logfile = os.path.join(self.log_dir,
-                                           self.subtest_name + "_" + \
-                                           self.server_log)
+                new_logfile = os.path.join(
+                    self.log_dir, self.subtest_name + "_" + self.server_log)
                 # rename on each of the servers
                 for host in self.hostlist_servers:
-                    subprocess.check_call(['ssh', host,
-                                           '[ -f \"{0}\" ] && '
-                                           ' mv \"{0}\" '
-                                           ' \"{1}\"'.format("/tmp/server.log",
-                                                             new_logfile)])
+                    subprocess.check_call(
+                        ['ssh', host,
+                         '[ -f \"{0}\" ] && mv \"{0}\" \"{1}\"'.format(
+                             SERVER_LOG, new_logfile)])
```

Comment:
Another candidate for `clush`, once we get that in place.

Reply:
Agreed